### PR TITLE
[Docs] Meshery Extensions Rename

### PR DESCRIPTION
**Pull request by vishalvivekm**
### Description

This PR is for MeshMap -> Kanvas

Exlcudes: 
- docs/_models ( controlled by integration spreadsheet, will need to meshmap -> kanvas there)
-  docs/_releases ( old release notes )
- docs/v0.4, docs/v0.5, docs/v0.6 ( older versions of docs )
- [docs/pages/extensions/snapshot/index.md](https://github.com/meshery/meshery/blob/master/docs/pages/extensions/snapshot/index.md) : refers to meshmap-snapshot gh action and repository e.g. `uses: layer5labs/MeshMap-Snapshot@v0.0.4`, which has not been renamed yet, and so references to "MeshMap Snapshot" remain unchanged for now.

**Notes for Reviewers**

- links like these https://playground.meshery.io/extension/meshmap?mode=design&design=ab35416d-7cf7-4540-8b2e-7271ffeadde2 need to change to have ~meshmap~ kanvas ?
Also, since these design links have been shared by users in past, it would be prudent to have redirection of sort in place from meshmap/* links to kanvas/*.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit b26aea5be22ef829bb5269ca8e839e04323c6d0e at: https://66e877372fbd3e299590e858--meshery-docs-preview.netlify.app

File: docs/pages/extensions/kanvas/edges-guide.md [hunk 1]
```diff
@@ -3,11 +3,11 @@ layout: default
 title: Edges Style Guide
 permalink: extensions/edges-shape-guide
 language: en
-abstract: Visualize and manage complex cloud-native relationships with MeshMap's dynamic edge styling in Kubernetes architecture.
+abstract: Visualize and manage complex cloud-native relationships with Kanvas's dynamic edge styling in Kubernetes architecture.
```
Comment (vishalvivekm) [inline docs/pages/extensions/kanvas/edges-guide.md:6]:
```suggestion
abstract: Visualize and manage complex cloud-native relationships with Kanvas' dynamic edge styling in Kubernetes architecture.
```

File: docs/pages/extensions/kanvas/index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+category: kanvas
+---
+
+# <img style="height: 4rem; width: 4rem;" src="{{site.baseurl}}/assets/img/kanvas-icon-color.svg" /> Kanvas <sup style="font-size: 1.8rem; vertical-align: top;">BETA</sup>
+
+Kanvas is a GitOps-infused cloud native <b>visual designer</b> for Kubernetes and cloud native applications. Kanvas works with your Kubernetes clusters so that you design and manage your infrastructure and applications between members of your team. Kanvas has two modes: <a href="https://layer5.io/cloud-native-management/kanvas/design">Designer</a> and <a href="https://layer5.io/cloud-native-management/kanvas/visualize">Visualizer</a>. Create and share your own Kubernetes deployments and patterns in Kanvas Designer. Deploy your designs and manage your infrastructure using Kanvas Visualizer. It includes multi-user collaboration features that allow teams to work together seamlessly towards achieving their goals in a more timely manner. Kanvas aims at removing all the challenging aspects of collaboration from the equation so that everyone on your team can be on the same page when it comes to managing cloud-native infrastructure projects.
+
+### Guide to use Kanvas's features
```
Comment (vishalvivekm) [inline docs/pages/extensions/kanvas/index.md:17]:
```suggestion
### Guide to use Kanvas' features
```

**Comment by l5io** [comment]
🚀 Preview for commit 50ea57b997c825846356a4b275be3498430c162e at: https://66e8791d92e93554e4cc277c--meshery-docs-preview.netlify.app

File: docs/pages/extensions/kanvas/publishing-a-design.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ language: en
 display-title: "false"
 list: include
 type: extensions
-category: meshmap
+category: kanvas
```
Comment (leecalcote) [inline docs/pages/extensions/kanvas/publishing-a-design.md:10]: add redirect from old permalink to new location?

File: docs/pages/extensions/kanvas/sharing-a-design.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ language: en
 display-title: "false"
 list: include
 type: extensions
-category: meshmap
+category: kanvas
```
Comment (leecalcote) [inline docs/pages/extensions/kanvas/sharing-a-design.md:10]: add redirect from old permalink to new location?

File: docs/pages/extensions/kanvas/tags-set.md [hunk 5]
```diff
@@ -7,14 +7,14 @@ abstract: Tags can be used to visually group components.
 display-title: "false"
 list: include
 type: extensions
-category: meshmap
+category: kanvas
```
Comment (leecalcote) [inline docs/pages/extensions/kanvas/tags-set.md:10]: add redirect from old permalink to new location?

**Comment by leecalcote** [comment]
Regarding your remarks under notes for reviewers, @vishalvivekm, good call on both of the bullet points. /extension/meshmap will change in the near future. When it does, but links within this repo's control and a redirection in that Provider's extension should be implemented.

**Comment by l5io** [comment]
🚀 Preview for commit ae6779ee05b26ab413573a23e36acf319acfea2e at: https://66e98b25cdef9a00a1edc5c0--meshery-docs-preview.netlify.app

**Comment by vishalvivekm** [comment]
> Regarding your remarks under notes for reviewers, @vishalvivekm, good call on both of the bullet points. /extension/meshmap will change in the near future. When it does, but links within this repo's control and a redirection in that Provider's extension should be implemented.

I’ve noted this ✍️, @leecalcote, and have also attended to your feedback.

**Review by leecalcote — APPROVED** [review]
Thank you

**Pull request merged**